modules/manage/partials/whole-cluster-restore.adoc

```diff
@@ -0,0 +1,183 @@
+
+=== Start a target cluster
+
+ifdef::env-kubernetes[]
+
+Deploy the target Redpanda cluster.
+
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    storage:
+      tiered:
+        <tiered-storage-settings>
+    config:
+      cluster:
+        cloud_storage_attempt_cluster_recovery_on_bootstrap: true
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+--
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`cluster-restore.yaml`
+[,yaml]
+----
+storage:
+  tiered:
+    <tiered-storage-settings>
+config:
+  cluster:
+    cloud_storage_attempt_cluster_recovery_on_bootstrap: true
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+--values cluster-restore.yaml
+```
+
+--set::
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --set storage.tiered.<tiered-storage-settings> \
+  --set config.cluster.cloud_storage_attempt_cluster_recovery_on_bootstrap=true
+```
+====
+--
+======
+
+- `storage.tiered`: Make sure to configure the target cluster with the same Tiered Storage settings as the failed source cluster.
+- `config.cluster.cloud_storage_attempt_cluster_recovery_on_bootstrap`: Automate cluster restore in Kubernetes. Setting to `true` is recommended when using an automated method for deployment. When bootstrapping a cluster with a given bucket, make sure that any previous cluster using the bucket is fully destroyed; otherwise Tiered Storage subsystems may interfere with each other.
+endif::[]
+
+ifndef::env-kubernetes[]
+For self-hosted, follow the steps to xref:deploy:deployment-option/self-hosted/manual/index.adoc[deploy a new cluster].
+
+For Redpanda Cloud, follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster].
+Cloud account credentials (such as secrets or access keys) are not included in backed up metadata. You must manually configure these credentials for the new cluster.
+
+NOTE: Make sure to configure the target cluster with the same Tiered Storage settings as the failed source cluster.
+endif::[]
+
+ifndef::env-kubernetes[]
+=== Restore to target cluster
+
+// Update xref when rpk command alias is implemented
+You can restore data from a failed cluster to a target cluster using the xref:reference:rpk/rpk-cluster/rpk-cluster-storage-recovery.adoc[`rpk cluster storage restore`] command. The wait flag `-w` tells the command to poll the status of the restore process and then exit when completed.
+
+[,bash]
+----
+rpk cluster storage restore start -w
+----
+
+=== Perform a rolling restart
+
+To ensure that the cluster reflects newly-restored configuration properties, perform a xref:manage:cluster-maintenance/rolling-restart.adoc[rolling restart] after the restore process.
```

Comment:
@andrwng a rolling restart shouldn't be required here, right?.

cc @kbatuigas

Reply:
Done ✅